helmfile.d/0120.cert-manager.yaml

```diff
@@ -0,0 +1,39 @@
+helmDefaults:
+  args:
+    - "--wait"
+    - "--recreate-pods"
+    - "--timeout=600" 
+    - "--force"
+    - "--reset-values"
+
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+
+releases:
+
+################################################################################
+## CERT-MANAGER - Automatic Let's Encrypt for Ingress  #########################
+################################################################################
+
+- name: "cert-manager"
+  namespace: "kube-system"
+  labels:
+    chart: "cert-manager"
+    component: "ingress"
+    namespace: "kube-system"
+    vendor: "jetstack"
```

Comment:
vendor: "kubernetes"

Reply:
No, the vendor is `jetstack`

Reply:
https://github.com/jetstack/cert-manager